SCClassLibrary/deprecated/3.9/osc/OSCresponder.sc

```diff
@@ -1,3 +1,27 @@
+//////////////////////////////////////////////////////////////////////////////
+//  _____________________________________
+// / HEY YOU! When you're removing this  \
+// | class, also make sure to remove the |
+// | "OSCresponder.respond" line from    |
+// \ Main:recvOSCmessage. Thanks!        /
+//  -------------------------------------
+// \                             .       .
+//  \                           / `.   .' "
+//   \                  .---.  <    > <    >  .---.
+//    \                 |    \  \ - ~ ~ - /  /    |
+//          _____          ..-~             ~-..-~
+//         |     |   \~~~\.'                    `./~~~/
+//        ---------   \__/                        \__/
+//       .'  O    \     /               /       \  "
+//      (_____,    `._.'               |         }  \/~~~/
+//       `----.          /       }     |        /    \__/
+//             `-.      |       /      |       /      `. ,~~|
+//                 ~-.__|      /_ - ~ ^|      /- _      `..-'
+//                      |     /        |     /     ~-.     `-. _  _  _
+//                      |_____|        |_____|         ~ - . _ _ _ _ _>
+//
+//////////////////////////////////////////////////////////////////////////////
```

Comment:
This should go into SC history as the nicest deprecation message ever. Maybe we should propose its use for the code style guide!